tools/asynq/cmd/task.go

```diff
@@ -53,6 +53,24 @@ func init() {
 	taskRunCmd.MarkFlagRequired("queue")
 	taskRunCmd.MarkFlagRequired("id")
 
+	taskCmd.AddCommand(taskEnqueueCmd)
+	taskEnqueueCmd.Flags().StringP("type_name", "t", "", "type name to enqueue the task as (required)")
+	taskEnqueueCmd.Flags().StringP("payload", "l", "", "payload to enqueue (required)")
+	// The following are the various OptionTypes; if not specified we won't pass them so that composeOptions()
+	// can apply its own defaults
+	taskEnqueueCmd.Flags().Int("retry", 0, "maximum retries")
```

Comment:
Parameter names are chosen to match the names at https://github.com/hibiken/asynq/blob/master/client.go#L67 .  The only exception is `id`; the `Option` is named `TaskID` but all of the other commands here use `id` so I figured it was better to stick with that.